criteria/bundle-policy-and-code.md

```diff
@@ -8,10 +8,10 @@ order: 2
 
 ## Requirements
 
-* A codebase MUST include the policy that the source code is based on.
-* A codebase MUST include all source code that the policy is based on, unless used for fraud detection.
+* A [codebase](../glossary.md#codebase) MUST include the policy that the source [code](../glossary.md#code) is based on.
+* A codebase MUST include all source code that the [policy](../glossary.md#policy) is based on, unless used for fraud detection.
```

Comment:
It could even go in the sentence before this if we want to be strict about the first occurrence.